modules/creating-logfilesmetricexporter.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-logfilesmetricexporter_{context}"]
+= Creating a LogFileMetricExporter resource
+
+In the {logging} version 5.8 and newer versions, the `LogFileMetricExporter` custom resource (CR) is no longer deployed with the collector by default. You must manually create a `LogFileMetricExporter` CR to collect container related metrics. If you do not create the `LogFileMetricExporter` CR, you may see a *No datapoints found* message in the {product-title} web console dashboard for *Produced Logs*.
```

Comment:
... to generate the logs produced metric of running containers....

It is only for this produced metric, not necessarily other container related metrics.